lib/screens/app/ecosystem/overview.dart

```diff
@@ -49,10 +66,6 @@ class _OverviewState extends State<Overview> {
     NavigationService.of(context).navigateTo(Routes.plantSeeds);
   }
 
-  void onBuy() {
-    NavigationService.of(context).navigateTo(Routes.buySeeds);
-  }
-
```

Comment:
That's cool (very cool) but we will actually rename everything buy to get - I will do that